### DISCUSSION TITLE
Building binary

### DISCUSSION THREAD

**Discussion by huitingyeong** [Checkmarx/kics#5718] [Q&A] [answered]
Hi i tried to run the commands to build v1.5.14 into a binary. (unable to run as docker in my environment)

I did:
```
git clone https://github.com/Checkmarx/kics.git
cd kics
make build
```
But the version output does not match the expected 1.5.14:
![image](https://user-images.githubusercontent.com/14289173/185583071-fb11468d-b48d-4da9-931c-d57bea2d27dc.png)

I see in Makefile the version is defined as `VERSION := snapshot-$(shell echo ${COMMIT} | cut -c1-8)`
Any way to make the version reflected as the number instead of commit value?

**Comment by cx-miguel-dasilva** [comment, marked as answer]
Hi @huitingyeong, hope you are doing great! 
Currently, in KICS we do not provide binaries, so the makefile is oriented to development purposes.
You can change the make file to get the result you want by changing the the `VERSION` variable to your liking.
